redbot/cogs/mod/names.py

```diff
@@ -91,7 +91,7 @@ class ModInfo(MixinMeta):
         special_date = datetime(2016, 1, 10, 6, 8, 4, 443000)
         is_special = user.id == 96130341705637888 and guild.id == 133049272517001216
 
-        roles = sorted(user.roles)[1:]
+        roles = sorted(user.roles, reverse=True)[:-1]
```

Comment:
I know it existed before, but this doesn't need sorting.

```suggestion
        roles = user.roles[-1:1:-1]
```
Since this is already sorted by d.py